site/content/docs/developer-guide/plugin-manifest.md

```diff
@@ -95,8 +95,8 @@ accessible to download from user’s machine. The relevant fields are:
 ## Specifying platform-specific instructions
 
 Krew makes it possible to install the same plugin on different operating systems
-(like `windows`, `darwin` (macOS), and `linux`) and different architectures
-(like `amd64`, `386`, `arm`).
+(e.g., Windows, macOS, and Linux) and different architectures
```

Comment:
we might need to restore this line as the `code` strings need to be preserved verbatim.